packages/sku/src/context/configSchema.ts

```diff
@@ -1,5 +1,6 @@
 import Validator from 'fastest-validator';
 
+// @ts-expect-error
```

Comment:
Can this have a reason, even just "Types not done yet" or something?

Reply:
We can either do this or workaround type it by using `as unknown as Validator.default` for the validator type. It fixes typescript typing and it still acts like it should.